### DISCUSSION TITLE
Knowledge Base (how to reduce chat request)

### DISCUSSION THREAD

**Discussion by casualmatt** [chatwoot/chatwoot#4608] [Feature Requests]
Hello to all, 

what i see missing in Chatwoot is a self service area for the customer that is looking for information, and to reduce the repetitive request a Knowledge Base addition to the chat widget could be a easy and simple fix to repetitive request. 

What you think?

**Comment by casualmatt** [comment]
just see in an [issue ](https://github.com/chatwoot/chatwoot/issues/4593) open 5h a go with the confirmation of this feature coming in to chatwoot.

**Reply by pranavrajs** [reply]
Yes, this is an upcoming feature.
